include/modules/meta/names.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
Maybe call this something like `types.hpp`. I think there is enough confusion between a module name and a module type.

Reply:
Renamed